### ISSUE TITLE
[Suggestion] Let Blood Magic altar use blood from other mods

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [WayofTime/BloodMagic#1793]
Such as https://www.curseforge.com/minecraft/mc-mods/evilcraft or Biomes o' Plenty's blood.

**Comment by Nebby1999** [comment]
Personally i doubt this will be a thing that's normally on in the config, since it could throw off the balance of the mod.

**Comment by wrincewind** [comment]
I'd say it'd be on those mods (or another) to provide compatibility. I'm not sure about Evilcraft's blood, but I know BoP blood is plentiful in the right parts of the nether, and having such a potent source would pretty much render a decent chunk of Blood Magic's progression (e.g. runes of sacrifice/self-sacrifice, well of suffering, incense altar, etcetera) redundant. You could just hook up a pump, get some acceleration/capacity/displacement runes going, and make whatever you want.

**Comment by recursiveSlacker** [comment]
"That isn't blood, it would have coagulated by now... don't ask how I know this."
The book seems to indicate it ain't exactly the same thing as life essence.

**Comment by WayofTime** [comment]
I echo Wrince's opinion on this. Know how Lava is super easy to get? That's just what will happen with LP if there was a 1:1 compatability.

**Comment by wrincewind** [comment]
Yeah, that was something I put in. To my mind, the act of spilling blood into the altar with the sacrificial dagger (amongst other methods) converts it into Life Essence, refining and purifying it into a usable form. It's why you need the altar and the structure around it - you can't just smear gore onto a bit of stone and hope for something to happen. Similarly, the process of filtering it into the Soul Network via a Blood Orb further refines it into LP for use in rituals and such. Each step is (barring some powerful rituals) irreversible. 

It might be more reasonable at a higher (configurable?) ratio, but that's a black hole of balancing issues that i wouldn't want to peer into. Is 10:1 too low? is 100:1 too high? Getting it to the point where it's *sometimes* a good idea and not *always* a good idea or *never* a good idea is ... shall we say, not my idea of a good time. :P

**Comment by SpoilersMC** [comment]
for the record- evilcraft allows 1:1 conversion of all other mod's blood to its own blood which is also 1:1 convertible to BM blood-all handled on it's end- at least this was the case in 1.7 & 1.12 - seeing as 1.16 is still in early dev im not sure if this is applicable _yet_ but knowing evilcraft the bound blood drop is definitely going to/does exist

**Comment by wrincewind** [comment]
Closing as 1.16 is out-of-support. (unlikely to be implemented by us in future versions - again, if other mods want to add this as a feature, they're welcome to)
